frontend/scss/components/molecules/pixi-status-intro.scss

```diff
@@ -59,7 +58,7 @@
     background-position: 60px 50px;
     background-color: color('white');
     pointer-events: none;
-    opacity: 0;
+    display: none;
     transition: opacity 0.2s ease-in-out, transform 0.3s ease-in-out;
```

Comment:
All the transitions have no effect anymore with this change, we should remove the related properties as well or think about toggling `aria-hidden` instead.

Reply:
Ah good catch. We can toggle `aria-hidden` instead if preferred.

Reply:
Depends on wether we want to keep fancyness 😁